packages/web-service/src/pages/permissions/common/permission-functions.js

```diff
@@ -84,3 +84,9 @@ export const getCheckYourAnswersData = async permissionData => {
   }
   return data
 }
+
+export const validateConditionalInput = (powerPlatformKeyValue, enteredValue, radioType) => {
```

Comment:
I think you can simplify to

powerPlatformKeyValue && !enteredValue) || (parseInt(radioType) === powerPlatformKeyValue